std/file.d

```diff
@@ -450,22 +459,112 @@ version (Windows) private void[] readImpl(scope const(char)[] name, scope const(
             OPEN_EXISTING, FILE_ATTRIBUTE_NORMAL | FILE_FLAG_SEQUENTIAL_SCAN,
             HANDLE.init);
     auto h = trustedCreateFileW(namez, defaults);
-
     cenforce(h != INVALID_HANDLE_VALUE, name, namez);
     scope(exit) cenforce(trustedCloseHandle(h), name, namez);
-    ulong fileSize = void;
-    cenforce(trustedGetFileSize(h, fileSize), name, namez);
-    size_t size = min(upTo, fileSize);
-    auto buf = () @trusted { return GC.malloc(size, GC.BlkAttr.NO_SCAN)[0 .. size]; } ();
+
+              ulong  fileSize     = void;
+    immutable bool   fileSizeOkay = trustedGetFileSize(h, fileSize);
+    immutable size_t maxSize      = fileSizeOkay ? min(fileSize, upTo) : upTo;
```

Comment:
I don't think vertical alignment like here is conformant to Phobos style. (Leading alignment is also at least indistinguishable from a mis-indented line.)

Reply:
Okay. It was merely a suggestion, because maybe other developers find it useful (if so, maybe the "style" is relaxed). But the lines can easily be re-indented.